backend/internal/service/oidc_service.go

```diff
@@ -1840,6 +1840,8 @@ func (s *OidcService) getUserClaimsFromAuthorizedClient(ctx context.Context, aut
 		claims["given_name"] = user.FirstName
 		claims["family_name"] = user.LastName
 		claims["name"] = user.FullName()
+		claims["display_name"] = user.DisplayName
+
 		claims["preferred_username"] = user.Username
 		claims["picture"] = common.EnvConfig.AppURL + "/api/users/" + user.ID + "/profile-picture.png"
```

Comment:
Suggest moving these profile claims *after* the custom claims, to avoid custom claims overriding them (see the point above about potential conflict on display_name)